tst/EnergyPlus/unit/HeatBalanceSurfaceManager.unit.cc

```diff
@@ -3126,4 +3127,355 @@ TEST_F(EnergyPlusFixture, HeatBalanceSurfaceManager_TestInitHBInterzoneWindow)
+    state->dataSurface->SurfWinGainConvShadeToZoneRep.allocate(3);
+    state->dataSurface->SurfWinOtherConvGainInsideFaceToZoneRep.allocate(3);
+    state->dataSurface->SurfWinGainIRShadeToZoneRep.allocate(3);
+    state->dataSurface->SurfWinFrameQRadOutAbs.allocate(3);
+    state->dataSurface->SurfWinFrameQRadInAbs.allocate(3);
+    state->dataSurface->SurfWinDividerQRadOutAbs.allocate(3);
+    state->dataSurface->SurfWinDividerQRadInAbs.allocate(3);
+    state->dataSurface->SurfWinHeatLossRep.allocate(3);
+    state->dataSurface->SurfWinGainConvGlazToZoneRep.allocate(3);
+    state->dataSurface->SurfWinGainIRGlazToZoneRep.allocate(3);
+    state->dataSurface->SurfWinLossSWZoneToOutWinRep.allocate(3);
+    state->dataSurface->SurfWinGainFrameDividerToZoneRep.allocate(3);
+    state->dataSurface->SurfWinGainConvGlazShadGapToZoneRep.allocate(3);
+    state->dataSurface->SurfWinGainConvShadeToZoneRep.allocate(3);
+    state->dataSurface->SurfWinOtherConvGainInsideFaceToZoneRep.allocate(3);
+    state->dataSurface->SurfWinGainIRShadeToZoneRep.allocate(3);
+    state->dataSurface->SurfWinFrameQRadOutAbs.allocate(3);
+    state->dataSurface->SurfWinFrameQRadInAbs.allocate(3);
+    state->dataSurface->SurfWinDividerQRadOutAbs.allocate(3);
+    state->dataSurface->SurfWinDividerQRadInAbs.allocate(3);
+    state->dataHeatBalSurf->SurfCTFConstInPart.allocate(3);
+    state->dataHeatBalSurf->SurfOpaqQRadSWInAbs.allocate(3);
+    state->dataHeatBalSurf->SurfQAdditionalHeatSourceInside.allocate(3);
+    state->dataHeatBalFanSys->QCoolingPanelSurf.allocate(3);
+    state->dataHeatBalFanSys->QRadSurfAFNDuct.allocate(3);
+    state->dataHeatBalSurf->SurfTempDiv.allocate(3);
+    state->dataHeatBalSurf->SurfIsOperatingPool.allocate(3);
+    state->dataHeatBalFanSys->PoolHeatTransCoefs.allocate(3);
+    state->dataHeatBalSurf->SurfIsOperatingPool.allocate(3);
+    state->dataHeatBalSurf->SurfTempTerm.allocate(3);
+    state->dataHeatBalSurf->SurfNetLWRadToSurf.allocate(3);
+    state->dataHeatBalSurf->SurfQSourceSinkHist.allocate(3);
+    state->dataHeatBalFanSys->QPoolSurfNumerator.allocate(3);
+    state->dataHeatBalSurf->SurfHcExt.allocate(3);
+    state->dataHeatBalSurf->SurfHSkyExt.allocate(3);
+    state->dataHeatBalSurf->SurfHGrdExt.allocate(3);
+    state->dataHeatBalSurf->SurfHAirExt.allocate(3);
+    state->dataHeatBalSurf->SurfTempInRep.allocate(3);
+    state->dataHeatBalSurf->SurfTempOut.allocate(3);
+    state->dataSurface->SurfWinHeatGainRepEnergy.allocate(3);
+    state->dataSurface->SurfWinHeatTransferRepEnergy.allocate(3);
+
+    state->dataHeatBal->ZoneMRT.allocate(1);
+
+    state->dataHeatBalSurf->SurfTempTerm(surfNum2) = 1.0;
+    state->dataHeatBalSurf->SurfQSourceSinkHist(surfNum2) = 1.0;
+    state->dataHeatBalSurf->SurfCTFConstInPart(surfNum2) = 1.0;
+    state->dataHeatBalFanSys->QPoolSurfNumerator(surfNum2) = 1.0;
+    state->dataHeatBalSurf->SurfTempOutHist(surfNum2) = 1.0;
+    state->dataHeatBalSurf->SurfTempDiv(surfNum2) = 1.0;
+
```

Comment:
Check `HeatBalanceSurfaceManager_TestSurfTempCalcHeatBalanceInsideSurfAirRefT`

Reply:
I didn't modify the contents. The function remains the same as the development branch.